packages/runtime/src/errors/errors-list.ts

```diff
@@ -207,9 +207,21 @@ by an index that does not exist.`
   },
   UNKOWN_OPCODE: {
     number: 31,
-    message: "Error encountered while parsing teal file unkown opcode %opcode% , Line : %line% ",
+    message: "Error encountered while parsing teal file: unkown opcode \"%opcode%\" for teal version #%version%, Line : %line% ",
     title: PARSE_ERROR,
     description: `Unkown Opcode encountered`
+  },
+  INVALID_MAX_COST: {
```

Comment:
can we rename  to `MAX_COST_EXCEEDED` or something else ?

Reply:
Ok, will change to `MAX_COST_EXCEEDED`